prowler/providers/aws/services/fms/fms_service.py

```diff
@@ -69,13 +69,16 @@ def __list_compliance_status__(self):
                     for fms_compliance_status in page.get(
                         "PolicyComplianceStatusList", []
                     ):
+                        compliance_status = ""
+                        if fms_compliance_status.get("EvaluationResults"):
+                            compliance_status = fms_compliance_status.get(
+                                "EvaluationResults"
+                            )[0].get("ComplianceStatus")
```

Comment:
are we handling the case that the compliance_status is None when the key ComplianceStatus does not exist?

Reply:
We are setting its default value just before the `if` block as `""`, we can set it to `None` (since in python `str` are falsy values)

Reply:
I'll do it and push it

Reply:
I mean in the check

Reply:
I'll write a test to see it